airflow/models/dagbag.py

```diff
@@ -107,7 +107,7 @@ def size(self):
 
     @property
     def dag_ids(self) -> List[str]:
-        return self.dags.keys()
+        return list(self.dags.keys())
```

Comment:
setf.dags.keys returns [KeysView](https://docs.python.org/3/library/collections.abc.html#collections.abc.KeysView). We want to have an non-llive object.